man/systemd.network.xml

```diff
@@ -2612,6 +2612,14 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>UseHopLimit=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When true, the hop limit received in the Router Advertisement will be
+          used. See <varname>IPv6HopLimit=</varname>. Defaults to true.</para>
```

Comment:
Now it sets per route hop limit, hence
- `will be used.` -> `will be set to routes configured based on the advertisement.`
- `See <varname>IPv6HopLimit=</varname>.` -> `See also <varname>IPv6HopLimit=</varname>.`